api/modules/users/views.py

```diff
@@ -166,7 +166,7 @@ def trip_friends_all(request):
     except Exception as e:
         return Response(str(e), status=status.HTTP_400_BAD_REQUEST)
 
-    serializer = UserSerializer(all_trips, many=True)
+    serializer = UserOnlySerializer(all_trips, many=True)
```

Comment:
Also since you are on it
Change the variable name from `all_trips` to `trip_friends`